### ISSUE TITLE
Segmented button is unlabelled in screenreader form field selections

### ISSUE THREAD

**Issue by lormanlau** [SAP/ui5-webcomponents#4000]
### **Bug Description**
The <ul> that has role="listbox" for the segmented button is missing accessible name (aria-labelledby) which is why its showing up as unlabelled in the JAWS Screenreader Form Field area.

### **Expected Behavior**
A clear and concise description of what you expect to happen.
Form Field shouldn't have unlabelled segmented button

### **Steps to Reproduce**
1. Launch JAWS 2020 screen reader
2. navigate to https://sap.github.io/ui5-webcomponents/playground/components/SegmentedButton/
3. press the key combination Insert + F5

### **Isolated Example**
If possible, please provide an isolated example by forking [CodeSandbox](https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html).
https://sap.github.io/ui5-webcomponents/playground/components/SegmentedButton/

### **Context**
- UI5 Web Components version: 1.0.0-rc.15
- OS/Platform: Windows
- Browser: Google Chrome 
- Affected component: Segmented button

### **Log Output / Stack Trace / Screenshots**
![image](https://user-images.githubusercontent.com/12117112/134735955-9bd7ffde-c270-4449-9d99-322dc8526313.png)

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SAP SuccessFactors
- Bussiness impact: Medium

**Comment by Todor-ads** [comment]
Hi colleagues from @SAP/ui5-webcomponents-topic-b,
can you please take a look at this issue.

Best regards,
Todor

**Comment by ilhan007** [comment]
Hello  @SAP/ui5-webcomponents-topic-b, @petyabegovska this is one of the ACC issues  by SF 
we need to complete by January. Raising the priority to High.

**Comment by ilhan007** [comment]
Hello @lormanlau the fix released - 0.31.27, you can use the "accessibleName" property to provide additional reading text.
